cfme/common/host_views.py

```diff
@@ -306,13 +309,13 @@ def is_displayed(self):
 
 class HostEditView(HostFormView):
     """View for editing a single host"""
+    breadcrumb = BreadCrumb()
     save_button = Button("Save")
     reset_button = Button("Reset")
-    change_stored_password = Text(".//a[contains(@ng-hide, 'bChangeStoredPassword')]")
 
     @property
     def is_displayed(self):
-        return self.in_compute_infrastructure_hosts and self.title.text == "Info/Settings"
+        return False
```

Comment:
better to return `False` as it can be False-positive - in case previous test got stuck on this view:

- for another host
- for this host as it happened in BZ 1584261